db/schema.rb

```diff
@@ -38,14 +38,23 @@
     t.integer  "judge_id"
     t.datetime "occur_at"
     t.text     "notes"
-    t.datetime "created_at",       null: false
-    t.datetime "updated_at",       null: false
-    t.integer  "region_id"
+    t.datetime "created_at",   null: false
```

Comment:
hmm - not sure how this got included but it's not an issue. If you do `git checkout master db/schema.rb` we can avoid merging schema changes along with this work.

Reply:
oops